content/integrations/streaming/datadog.textile

```diff
@@ -0,0 +1,54 @@
+---
+title: Datadog
+meta_description: "Connect Ably and Datadog to monitor messages, channels, and connections in realtime, integrating your Ably statistics with your existing Datadog setup."
+meta_keywords: "Datadog, integrations, statistics, metrics, monitoring, analytics, enterprise"
+---
+
+The Ably "Datadog":https://docs.datadoghq.com/integrations/ably/ integration enables you to monitor your application's statistics. Every 60 seconds, Ably streams a comprehensive set of "statistics":/docs/metadata-stats/stats#metrics to the Datadog API using the a generated API key, which is derived from "metadata subscriptions.":/metadata-stats/metadata/subscribe
+
+h2(#setup). Setup the Datadog integration
+
+To connect Ably with Datadog, you must "request":https://ably.com/support access to the integration. Once granted, you can authorize the integration through Datadog's "OAuth":https://docs.datadoghq.com/developers/integrations/oauth_for_integrations/ flow. This process requires the @api_keys_write@ scope, allowing Ably to push data to your Datadog account.
+
+Once the integration is active, Datadog provides a specific Ably "dashboard":https://docs.datadoghq.com/integrations/ably/, allowing you to monitor key metrics without extra setup.
+
+The following steps setup the Datadog integration:
+
+# In Datadog, go to *Integrations*, find the *Ably* tile, and click *Install Integration*.
+# Click *Connect Accounts* to start the authorization process. Datadog redirects you to Ably.
+# Log in to your *Ably* account.
+# Select your application from the *Your Apps* page.
+# Navigate to *Integrations*, and select *Connect to Datadog*.
+# Datadog authorization page, authorize Ably using *OAuth* to grant access. The required authorization scope is: @api_keys_write@.
+# After completing authorization, you will be redirected to the *Ably dashboard*, and the process is complete.
+
+h2(#remove). Remove access
+
+Removing access disconnects Ably from Datadog, stopping data transmission and revoking authorization. Follow the steps remove the Ably and Datadog integration using either platform.
+
+h3(#in-ably). Remove access using Ably
+
+* Open your application's integration settings.
+* Click *Remove* next to the Datadog integration.
+* Ably revokes OAuth credentials and stops metric transmission.
+
+h3(#in-datadog). Remove access using Datadog
+
+* Remove associated Ably API keys via *Integrations* or *API Keys*.
+* Adjust scopes or entirely revoke OAuth tokens if necessary.
+
+h2(#lite). Datadog lite
+
+Ably streams a limited set of "statistics":/docs/metadata-stats/stats#metrics to the Datadog API. Datadog-lite minimizes data usage, making it a cost-effective alternative for use cases that don't need full statistics.
```

Comment:
I think this needs an introduction as to what Datadog lite is, e.g. 'Datadog Lite is...' 

I think we should also mention which packages that it's available on, rather than referring to it as a cost-effective alternative.